source/index.html.md

```diff
@@ -201,6 +203,7 @@ error categories, the last two digits define specific errors.
 ### Operation Errors: 11 (500 error)
 * 1101: Error when processing the withdrawal
 * 1102: Error registering callback URL
+* 1103: Deprecated functionality
```

Comment:
Deprecated functionality is not a 5xx errors - HTTP 5xx errors are server errors - the client using something we have deprecated isn't a server problem.

Reply:
Fixed, thanks for spotting  it.